modern/src/map/GeofenceMap.js

```diff
@@ -4,6 +4,15 @@ import { useEffect, useState } from 'react';
 import { map } from './Map';
 import { useEffectAsync } from '../reactHelper';
 import { reverseCoordinates } from './mapUtil';
+import circle from '@turf/circle';
+
+
+const circleToPolygon = (item) => {
+  let coord = item.area.replace(/CIRCLE|\(|\)|,/g, " ").trim().split(/ +/);
+  var options = {steps: 64, units: 'meters'};
+  let c = circle([Number(coord[1]), Number(coord[0])], Number(coord[2]), options);
```

Comment:
Please use full words for variable names. Maybe `polygon` or something here.

Reply:
ok